test/coverage/communication/PublisherTest.cpp

```diff
@@ -162,4 +165,36 @@ TEST_F(TestPublisher, PublishSuccessWithoutPriority) {
 	          UPriority::UPRIORITY_CS1);
 }
 
+TEST_F(TestPublisher, PubSubSuccess) {
+	// sub
+	auto transportSub =
+	    std::make_shared<uprotocol::test::UTransportMock>(source_);
+
+	auto callback = [this](auto arg1) {
+		return this->handleCallbackMessage(arg1);
+	};
```

Comment:
Minor: the message could just be captured directly here instead of adding a member to the test class

```suggestion
    uprotocol::v1::UMessage captured_message
	auto callback = [&captured_message](auto message) {
		captured_message = message;
	};
```

Reply:
Done